src/main/java/gui/misc/TransitionHelper.java

```diff
@@ -25,20 +27,36 @@
 
     @Getter
     private Node node;
+    @Getter
+    private ArrayList<EventHandler> addedHandlers;
```

Comment:
Use the interface that you want to use, as opposed to the implementing class. `List` in this case.